Tasks/app-store-release/task.json

```diff
@@ -219,6 +219,15 @@
             "groupName": "releaseOptions",
             "visibleRule": "releaseTrack = TestFlight"
         },
+        {
+            "name": "appSpecificId",
+            "type": "string",
+            "label": "App specific apple Id",
```

Comment:
Could you please also rename this label to `App Specific Apple Id`?  Other changes LGTM.

Reply:
Sure, thanks 👍 Fixed